examples/ccvm_boxqp_mf.py

```diff
@@ -36,9 +35,9 @@
         boxqp_instance.scale_coefs(solver.get_scaling_factor(boxqp_instance.q_matrix))
 
         # Solve the problem
-        solution = solver.solve(
-            instance=boxqp_instance,
-            post_processor=None,
+        solution = solver(
+            instance=boxqp_instance, # solve_type = "Adam",
+            post_processor=None, # hyperparameters=dict(beta1=0.9, beta2=0.999, alpha=0.001),
```

Comment:
Are these intending to stay commented out?
Either way is fine with me, I just wanted to flag it.

Reply:
To make it consistent, all demo scripts use the "adam" solver.